packages/atst/package.json

```diff
@@ -26,13 +26,13 @@
   "scripts": {
     "dev": "tsx src/cli.ts",
     "clean": "rm -rf ./node_modules && rm -rf ./dist && rm -rf ./coverage",
-    "build": "yarn tsup",
+    "build": "tsup",
     "generate": "wagmi generate",
-    "lint": "yarn lint:fix && yarn lint:check",
+    "lint": "pnpm lint:fix && pnpm lint:check",
     "lint:check": "eslint . --max-warnings=0",
-    "lint:fix": "yarn lint:check --fix",
+    "lint:fix": "pnpm lint:check --fix",
     "test": "vitest",
-    "typecheck": "yarn tsc --noEmit"
+    "typecheck": "tsc --noEmit"
```

Comment:
in pnpm (and yarn and npm) you don't need to type yarn or pnpm first to run a node_module script